internal/backends/winit/glwindow.rs

```diff
@@ -227,15 +229,19 @@ impl<Renderer: WinitCompatibleRenderer + 'static> WinitWindow for GLWindow<Rende
     }
 
     fn resize_event(&self, size: winit::dpi::PhysicalSize<u32>) {
-        if let Some(_) = self.borrow_mapped_window() {
-            // NOTE: slint::Window::set_size will call set_size() on this type, which calls
-            // set_inner_size on the winit Window. On Windows that triggers an new resize event.
-            // Therefore we call slint::Window::set_size within the borrow, so that the set_size's
-            // borrow in this type fails.
-            let physical_size = physical_size_to_slint(&size);
-            self.window.set_size(physical_size);
-            self.renderer.resize_event(physical_size);
-        }
+        // slint::Window::set_size will call set_size() on this type, which would call
+        // set_inner_size on the winit Window. On Windows that triggers an new resize event
+        // in the next event loop iteration for mysterious reasons, with slightly different sizes.
+        // I suspect a bug in the way the frame decorations are added and subtracted from the size
+        // we provide.
+        // Work around it with this guard that prevents us from calling set_inner_size again.
+        assert!(!self.in_resize_event.get());
+        self.in_resize_event.set(true);
+        scopeguard::defer! { self.in_resize_event.set(false); }
```

Comment:
this is a bit weak to introduce a dependence just for that.  but that's a light dependency.

Reply:
I can also do it without - no strong preference. Let me know what you prefer.

Reply:
I think i prefer without so we don't add too many deps - no strong preference

Reply:
As a side-note: The reason why I used scopeguard is because it's already a dependency of corelib.

Reply:
keep it then